Create adapt-to-large-screens (#9946)
flutter/website: src/ui/layout/responsive/adapt-to-large-screens.md

---
title: Adapt app to large screens
description: Best practices for developing Flutter apps on large device formats
short-title: Adapt to large screens
---

{% include docs/yt_shims.liquid %}

You can optimize  your app to improve its look and feel on large screens. 
Flutter defines [large screens][] as tablets, foldables, and ChromeOS devices 
running Android. Demand for large screens continues to increase. 
As of December 2023, more than [270 million active large screen][] and 
foldable devices run on Android.

## Why does this matter for developers?

> When your app supports large screens, they will also receive the
> following benefits.
> 
> * Increases your business metrics. They would show increased user 
>   engagement on tablets for apps with layouts optimized to fill the full screen.
> * Improves the [technical quality][] to increase visibility for your
>   app in the Play Store on these devices. Recent [Play Store updates][] show 
>   ratings by device type and notify users when an app lacks large screen support. 

Reviewer: Are there benefits to iOS apps, as well?

```suggestion
When your app supports large screens, it also receives the
following benefits:

* Improves your app's metrics to show increased user engagement
  when optimized to fill the screen.
* Increases your app's visibility in the Play Store.
  Recent [Play Store updates][] show ratings by device type and
  indicates when an app lacks large screen support. 
```
